src/ages/distributions/UsersDistribution.ts

```diff
@@ -1,6 +1,7 @@
 export interface UsersDistribution {
   age: string;
   epoch: string;
```

Comment:
Why keeping `age` and `epoch` here?

Reply:
It's used for readability only so I think yes

Reply:
But for future ages, we'll have `{ age: 'age4', epoch: 'epoch10' }` then `{ age: 'age5', epoch: 'epoch11' }`, etc.? It's a little redundant no? Unless we consider we could seamlessly use `age` and `epoch`?

Reply:
always epoch1 for age with one epoch

Reply:
Let's discuss about that later 👌🏼